src/prompt.py

```diff
@@ -1,35 +1,42 @@
-from colorama import init, Fore, Style
-
-init(autoreset=True)
+from rich import print as rich_print
+from src.console import console
 
 
 def on_watch_start(exercise_path):
-    print(f"{Style.BRIGHT}🤖🤖🤖 Watch mode started.")
-    print(f"You can start to work on exercise {exercise_path}.\n")
+    rich_print("[bold]:robot::robot::robot: Watch mode started.[/bold]")
+    rich_print(f"You can start to work on exercise {exercise_path}.\n")
```

Comment:
Could be useful to specify colors variable in a separate file.
We could later use different color schemes for core curriculum vs diploma awarding tracks.

Reply:
sure, this is a `good enough` solution for the current need, but we will later bring some print hierarchy with color schemes